### DISCUSSION TITLE
What is the plan for Open WebUI "store"? I have an idea

### DISCUSSION THREAD

**Discussion by Darin755** [open-webui/open-webui#1310] [Ideas]
So I really like the idea of having what is effectively a store for Modelfiles, prompts and maybe themes. Right now I see that the Open webUI web page has a store front https://openwebui.com/

I was thinking, this could be improved in many ways. What I would first do is get rid of Google Analytics and all of the other extras as they are overwhelming and make the web page unnecessarily heavy. You could move the store front itself to a sub domain so that you could have more flexibility. I like the Ollama website but you also could model off of F-droid or Flathub.

For the backend, you could allow anyone to host there own repo by standardizing the repo format and creating guides on how to make alternative store fronts. I would create a UI within Open WebUI for searching though content. Searching should happen locally and the index for all packages could be automatically or manually refreshed from time to time. If this sounds like a Linux package manager or F-droid then you would be right. The package manager architecture is well proven and wuld allow for a lot of flexibility. 

I am totally willing to work on making this happen but it is kind of pointless if I try to work on it without the support of the developers of Open webUI.

**Comment by justinh-rahb** [comment]
Not that I disagree necessarily with putting more work into openwebui.com... what you propose would potentially be even bigger than the WebUI itself, which needs to remain our main focus right now.

**Reply by Darin755** [reply]
Theoretically you could offload some of the work to the community by allowing them to manage the content.

A get your point though. Thanks for all the work the devs put into making Open webUI. I appreciate what you do.

**Reply by justinh-rahb** [reply]
Not that we haven't got grander ambitions mind you 😉 And the community is essential to it becoming reality.
